packages/web/src/common/store/account/sagas.js

```diff
@@ -182,34 +181,30 @@ function* onSignedIn({ payload: { account, isSignUp = false } }) {
   }
 }
 
-export function* fetchAccountAsync({ fromSource = false, isSignUp = false }) {
+export function* fetchAccountAsync({ isSignUp = false }) {
   const audiusBackendInstance = yield getContext('audiusBackendInstance')
   const remoteConfigInstance = yield getContext('remoteConfigInstance')
-  const localStorage = yield getContext('localStorage')
   const isNativeMobile = yield getContext('isNativeMobile')
   const isElectron = yield getContext('isElectron')
   const fingerprintClient = yield getContext('fingerprintClient')
 
   yield put(accountActions.fetchAccountRequested())
 
-  if (!fromSource) {
-    yield fetchLocalAccountAsync()
```

Comment:
So we don't need to call this anymore?

Reply:
Yeah discussed with @amendelsohn  a bit -- I can't find the utility of this b/c we fetchLocal account twice on startup currently

Reply:
It also didn't look like `fromSource` was ever true, but possible we missed something

Reply:
I couldn't find any instances of it being true either. But was worried since we are doing the local fetch in the false case. But if we are already doing fetchLocal as part of the `getAccount` logic we should be good

Reply:
It was never called with true lol!